imports/plugins/custom/gdr-landing-page/client/styles/main.less

```diff
@@ -522,12 +522,20 @@
 }
 
 .slick-arrow {
- background-color: transparent;
+ color: purple !important;
+ background-color: #6F4C7F;
+ opacity: 0.3;
 }
 
 .slick-arrow:hover,
+.slick-arrow:focus {
+  background-color: purple !important;
+  color: snow;
+  opacity: 1;
+}
+
 .slick-dots {
-  background-color: #E9E9E9
+  background-color: #e9e9e9;
```

Comment:
We should use less variables for our colors so they are reusable and consistent. Can you please use less variables here?